docs/get-started/sumo-logic-ui-new.md

```diff
@@ -0,0 +1,225 @@
+---
+id: sumo-logic-ui-new
+title: Tour the Sumo Logic UI
```

Comment:
Does this need a different title, like "Tour the New Sumo Logic UI"?

Reply:
good catch